README.md

```diff
@@ -105,16 +105,41 @@ The general idea of portals is summed up above: inset pre-rendering, activation,
 
 ### Privacy threat model and restrictions
 
+The Portals design is intended to comply with the [W3C Target Privacy Threat Model](https://w3cping.github.io/privacy-threat-model/). This section discusses the aspects of that threat model that are particularly relevant to Portals and how the design satisfies them.
+
+A portal can contain either a same-site or cross-site resource. Same-site portals don't present any privacy risks, but cross-site resources risk enabling [cross-site recognition](https://w3cping.github.io/privacy-threat-model/#model-cross-site-recognition) by creating a messaging channel across otherwise-partitioned domains.
+
+This design assumes that storage will be [partitioned](https://github.com/privacycg/storage-partitioning), even though that is not the status quo in all browsers. Because portaled documents can be activated into a top-level browsing context, they (eventually) live in the first-party storage partition of their origin. This means we have to prevent communication with the surrounding document to the same extent we prevent it with a cross-site link opened in a new tab.
+
+#### Channels that are blocked
```

Comment:
This section contains both channels that are blocked for both for cross-origin portals (postMessage, credentials/storage, ...) and things which we decided to make the same everywhere (most notably initial size) for simplicity. It's probably worth signaling which category each restriction is in. Maybe with a blanket statement plus a sentence noting that portal resizing/initial sizing is an exception?

Reply:
I've mentioned cross-origin or same-origin explicitly in each case, since neither is overwhelmingly common. None of these restrictions are currently planned to just be cross-site, right?

Reply:
> one of these restrictions are currently planned to just be cross-site, right?

Correct.

Reply:
The clarifications here look great, thank you!